src/test/java/com/pingcap/tikv/meta/TiSelectRequestTest.java

```diff
@@ -53,6 +54,9 @@ private static TiTableInfo createTable() {
 
   @Test
   public void testSerializable() throws Exception {
+    if (true) {
```

Comment:
What is this?

Reply:
Sorry for the confusing code, originally I intended to skip this test but forgot to bring it back to work. Fixed with new `TiDAGRequestTest`.